src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -450,15 +456,20 @@ private JMenu createToolsMenu() {
 		menu.addSeparator();
 		
 		TurtleModifierAction a4 = new FlipTurtleAction(1,-1,Translator.get("FlipH"));
+		//a4.putValue(Action.ACCELERATOR_KEY, KeyStroke.getKeyStroke(java.awt.event.KeyEvent.VK_H, java.awt.event.InputEvent.SHIFT_DOWN_MASK));//"shift H" //shift should never been an accelerator key
```

Comment:
it is dead code, isn't it ? 
If yes, you should remove it :)

Reply:
Ha yes it has become dead ... the time I find another less problematic keyboard shortcuts.
Thanks this comment got me out of procrastination. :)